.nvmrc

```
@@ -0,0 +1 @@
+14.19.3
```

```suggestion
18.16.1
```

Shocked that this even had v14 back in 2022, when v14 was already on maintenance LTS. At any rate, it's EOL now.

That said, I'll leave it up to Krisk to determine if he wants to merge this but personally I'd recommend against it because at that point you should also support other node version managers such as [volta.sh](https://volta.sh) and I would recommend against opening that can of worms and just letting developers deal with this themselves. If anything `.nvmrc` should be added to `.gitignore`.